terraform/install.sh

```diff
@@ -112,7 +112,14 @@ promptForProject() {
           IFS=$' |'
           opt=($opt)
           project_id=${opt[0]}
-          break
+          # check if this project has a bucket
+          bucket_name="$project_id-bucket"
+          gcloud config set project "$project_id"
+          if [[ -n "$(gsutil ls | grep gs://$bucket_name/)" ]]; then
+            break
+          else
+            log "This is an outdated project that doesn't support GCS. Please try to use a new one."
```

Comment:
This project uses old version of Sandbox not compatible with stateful provisioning. Please create a new project instead.